crates/bevy_ecs/src/bundle.rs

```diff
@@ -8,59 +8,34 @@ use crate::{
 use bevy_ecs_macros::all_tuples;
 use std::{any::TypeId, collections::HashMap};
 
-/// A dynamically typed ordered collection of components
+/// An ordered collection of components
 ///
 /// See [Bundle]
 /// # Safety
 /// [DynamicBundle::type_info] must return the TypeInfo for each component type in the bundle, in the _exact_
```

Comment:
This still refers to `DynamicBundle`.

Reply:
Haha sorry that was sloppy. We should have ci check for doc comment warnings.